resources/sass/default.style.scss

```diff
@@ -7,3 +7,4 @@
 @import "components/patterns";
 @import "components/blocks";
 @import "components/agenda";
+@import "components/modal_page";
```

Comment:
So, this scss will always be included, even when modal_page is not installed or no modal on the current page?

An alternative would be to generate a separate asset for this.
But, given that scss is so small, maybe not worth it.